src/styles/blocks/page.scss

```diff
@@ -0,0 +1,23 @@
+.page {
+  &__cards-container {
+    display: flex;
+    max-width: getMaxWidth($min-padding);
+    margin: auto;
+    padding: 50px $min-padding 72px;
+    flex-wrap: wrap;
+
+    @media (min-width:1200px) {
```

Comment:
we have a requirement:
there should always be 4 cards in a row (not 2, 3 or 5)

So now, no needs to use media-quueries

Reply:
Hello! Thank you for your comments! Could you please help me? I've been racking my brain but cannot find out: how to keep the page structure when removing @media and body tag (the very last comment)? Where is my mistake? Thank you!

Reply:
Hello! @Viktor-Kost, I apologize for bothering you again. Could you please check my message above? Thank you!